runtime/common/src/lib.rs

```diff
@@ -52,13 +52,22 @@ macro_rules! decl_common_types {
 
         type Address = sp_runtime::MultiAddress<AccountId, ()>;
 
+        #[cfg(feature = "with-global-disputes")]
```

Comment:
Is there a reason we still have this feature here? I thought we wanted to get rid of it.

Reply:
It is unfortunately only removed for the court-overhaul branch, but not for release-dispute-system. So, this is a little tricky.. We could wait for courts merge and then resolve conflicts with this branch.

Reply:
fixed by that https://github.com/zeitgeistpm/zeitgeist/pull/912/commits/77b88253fa88ac02cfc3e187ba6a52d024680e08 and this https://github.com/zeitgeistpm/zeitgeist/pull/912/commits/d1b437de0996c069d86317e2c80faf532d500e6e

Reply:
<img width="819" alt="Screenshot 2023-08-23 at 11 06 40" src="https://github.com/zeitgeistpm/zeitgeist/assets/49552287/dde494cb-0286-43df-8fe9-425fe85d9975">

"I ain't readin' all that."